EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -11382,13 +11382,13 @@ QUEST_LV_0100_20151224_011374	You have my utmost gratitude.{nl}Right before I lo
 QUEST_LV_0100_20151224_011375	Wha...? You already had it on you?{nl}That is great news. Bring it to the upper side of Amesti Uphill and sprinkle the energy around.{nl}
 QUEST_LV_0100_20151224_011376	I believe that utilizing the malicious force of the monsters would be a good idea. {nl}Use it on monsters to purify their energies and absorb them.
 QUEST_LV_0100_20151224_011377	By talking with the Chief Owl Sculpture, I got the feeling we could work this out.{nl}The demons are using the evil energy to corrupt Letas Stream and to trap souls.{nl}
-QUEST_LV_0100_20151224_011378	You will find the evil energy concentrated on the Svaigul Hill.{nl}Set up Namott of Suppression around it to remove the evil energy.
-QUEST_LV_0100_20151224_011379	Stopping the spot where the energy of the forest is getting drained is an urgent matter. {nl}The spot is on the Sevyni Yard.
-QUEST_LV_0100_20151224_011380	Without the Black Spirit, the demon installation will become weak.{nl}But the surveilance spheres are guarding the Sevyni Yard. {nl}Approaching it won't be a cake walk.{nl}
+QUEST_LV_0100_20151224_011378	You will find the evil energy concentrated on the Svaigulio Hill.{nl}Set up Namott of Suppression around it to remove the evil energy.
+QUEST_LV_0100_20151224_011379	Stopping the spot where the energy of the forest is getting drained is an urgent matter. {nl}The spot is on the Senyvas Yard.

```

Comment:
`Stopping the spot where the energy of the forest is getting drained is an urgent matter.`
to
`Our priority is stopping the spot where the energy of the forest is getting drained.` ?